setup.py

```diff
@@ -155,9 +155,14 @@
 ]
 extras_require = {
     ':sys_platform != "win32"': ['terminado>=0.3.3'],
-    'doc': ['Sphinx>=1.1'],
+    'doc': [
+        'Sphinx>=1.1',
```

Comment:
Is this already available on readthedocs?
Last time I checked it was at 1.3.5 ...